src/App.vue

```diff
@@ -24,7 +24,7 @@ export default {
     showPosition() {
       if(navigator.geolocation) {
         navigator.geolocation.getCurrentPosition(position => {
-          this.geolocation = position;
+          this.$store.commit('updateGeolocation', position)
```

Comment:
Really enjoy how simple it is to work with Vuex! Makes it really simple to replace areas of the App that were giving us trouble before and make them global.